playbooks/ci_all_machines_tests.yaml

```diff
@@ -23,4 +23,21 @@
         dest: >-
           {{ cukinia_test_prefix | default('..') }}/cukinia_{{ inventory_hostname
           }}.xml
+        flat: yes       
+
+- hosts: "{{ machines_tested | default('cluster_machines') }}"
```

Comment:
Is it not better to apply that on "cluster_machines" only ?
Otherwise, you can target machine tested without VMs with
`hosts: "{{ (machines_tested | default(groups['cluster_machines'])) | difference(groups['VMs']) }}"`